sci-libs/opencascade/opencascade-7.7.2.ebuild

```diff
@@ -32,12 +32,11 @@ fi
 
 LICENSE="|| ( Open-CASCADE-LGPL-2.1-Exception-1.0 LGPL-2.1 )"
 SLOT="0/$(ver_cut 1-2)"
-IUSE="X debug doc examples ffmpeg freeimage freetype gles2-only gui json +opengl optimize tbb test testprograms tk vtk"
+IUSE="X debug doc examples ffmpeg freeimage freetype gles2-only inspector json +opengl optimize tbb test testprograms tk vtk"
```

Comment:
This one will have to be revbumped

Reply:
I'd really like to avoid causing a rebuild of opencascade-7.7.2 for that, as that only exists for freecad. There were already complaints about "unneccessary" rebuilds.

The option to build the inspector was only added with 7.7.2/7.8.0. Anyone that actually wants this is likely to be on 7.8.0 and gets this for free with the 7.8.1 bump. Also the on disk files don't change with this.

Reply:
> Also the on disk files don't change with this.

The removal of `-DUSE_QT="$(usex gui)"` does not change the installed cmake files?  I thought removing this option would resolve the configure problem in `freecad[qt6]`.

> I'd really like to avoid causing a rebuild of opencascade-7.7.2 for that, as that only exists for freecad. There were already complaints about "unneccessary" rebuilds.

Folks always complain about this, but there is a reason the policy is to revbump anyway. If we do not revbump then users will still run into the problem with freecad that we are fixing here. The fact that this version is kept around specifically for freecad is precisely why it should be revbumped.

Reply:
Turned it into a revbump.